### ISSUE TITLE
Load infrantry in APC on one side only

### ISSUE THREAD

**Issue by yaaaaa** [OpenRA/OpenRA#10999]
![jqueeevql7u 1](https://cloud.githubusercontent.com/assets/11130958/14105254/3d4a408e-f5cc-11e5-9dce-7f2e8ec13c1d.jpg)
In original game 

But in OpenRA it happens from all sides.

Issue? Or it is done so on purpose? and if so, how can we return the loading units, with only one side?

**Comment by HenrytheSlav** [comment]
Well, it is slower and less practical. Is there any reason except staying true to the original for this?

**Comment by pchote** [comment]
This was an intentional change, because the original transport behavior was needlessly horrible.
If somebody wanted to use the original behaviour they would need to add new code for it in the Cargo trait.

**Comment by yaaaaa** [comment]
PassengerFacing in Cargo trait?

**Comment by pchote** [comment]
No. None of the existing properties on Cargo, because Cargo doesn't support it.

**Comment by yaaaaa** [comment]
which code I must add in Cargo?

**Comment by ScottNZ** [comment]
I think the current behaviour suits OpenRA gameplay much better.
